production/ksonnet/promtail/config.libsonnet

```
@@ -10,7 +10,7 @@
         username:: '',
         password:: '',
         scheme:: 'https',
-        hostname:: 'logs-us-west1.grafana.net',
+        hostname:: 'set valid hostname here',
```

This is what you are looking for. It could be a breaking change though for people updating their lib and promtails suddenly stopping to work.
```suggestion
        hostname:: error 'must define a valid hostname',
```

---

Thanks @sandeepsukhani - that looks perfect!

---

> It could be a breaking change though for people updating their lib and promtails suddenly stopping to work.

@sandeepsukhani If people are using this default setting, all they're getting is `404`s, so it's not working for them _anyway_ 😉